platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -41,6 +39,8 @@
 #include "main/main.h"
 #include "scene/resources/texture.h"
 
+#include "display_server_x11.h"
```

Comment:
I wonder why that move is needed, sounds to me like we have some includes above that need to go to the header.

Reply:
I would prefer we fix this properly, as someone might move this back up following our includes policy and trigger the same error.

I haven't tested but at least this might help:
```diff
diff --git a/platform/linuxbsd/x11/display_server_x11.cpp b/platform/linuxbsd/x11/display_server_x11.cpp
index 4d3eecd4fc..896b7b95eb 100644
--- a/platform/linuxbsd/x11/display_server_x11.cpp
+++ b/platform/linuxbsd/x11/display_server_x11.cpp
@@ -28,6 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
+#include "display_server_x11.h"
+
 #ifdef X11_ENABLED
 
 #include "core/config/project_settings.h"
@@ -39,8 +41,6 @@
 #include "main/main.h"
 #include "scene/resources/texture.h"
 
-#include "display_server_x11.h"
-
 #if defined(VULKAN_ENABLED)
 #include "servers/rendering/renderer_rd/renderer_compositor_rd.h"
 #endif
diff --git a/platform/linuxbsd/x11/display_server_x11.h b/platform/linuxbsd/x11/display_server_x11.h
index 1eba2f8a4c..dbe8a0ce2b 100644
--- a/platform/linuxbsd/x11/display_server_x11.h
+++ b/platform/linuxbsd/x11/display_server_x11.h
@@ -36,6 +36,8 @@
 #include "servers/display_server.h"
 
 #include "core/input/input.h"
+#include "core/os/mutex.h"
+#include "core/os/thread.h"
 #include "core/templates/local_vector.h"
 #include "drivers/alsa/audio_driver_alsa.h"
 #include "drivers/alsamidi/midi_driver_alsamidi.h"
```

Reply:
FWIW it still builds fine for me with this applied, thanks!